Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/edge-data-store.md

# Edge Data Store

> Edge Data Store (EDS) is a lightweight data collection and storage application designed to capture data at the edge of networks for historical storage and analysis. It runs on small, rugged devices or embedded in existing industrial hardware, and is designed to be resilient and require minimal installation and administration. While not a replacement for a PI System or AVEVA Data Hub, EDS augments the PI System and AVEVA Data Hub by collecting and storing data in situations where deploying a full system is impractical.

Reviewer: ```suggestion
Edge Data Store (EDS) is a lightweight data collection and storage application designed to capture data at the edge of networks for historical storage and analysis. It runs on small, rugged devices or embedded in existing industrial hardware and is designed to be resilient and require minimal installation and administration. While not a replacement for a PI System or AVEVA Data Hub, EDS augments the PI System and AVEVA Data Hub by collecting and storing data in situations where deploying a full system is impractical.
```

* Custom application using REST API

> Once collected, the data is stored locally in configurable data storage within EDS, until it can be sent to permanent storage in a PI System or in AVEVA Data Hub through periodic egress. The data can also be read from local storage by custom applications using REST APIs.

Reviewer: ```suggestion
Once collected, EDS stores the data locally in configurable data storage until it can be sent to permanent storage in a PI System or in AVEVA Data Hub through periodic egress. Custom applications that use REST APIs can also be read from local storage.
```

* Data Storage – Stores data locally until it can be egressed

* Data egress – Sends data from storage to PI Server or AVEVA Data Hub

>   **Note:** OSIsoft Cloud Services is also supported as an egress destination.

Reviewer: ```suggestion
  **Note:** EDS also supports OSIsoft Cloud Services as an egress destination.
```

* Configuration – Use REST or the EdgeCmd tool to configure EDS as a whole or each component individually and to view the current configuration

> EDS requires an endpoint to connect to REST APIs on the local device, which is shown outlined in blue in the diagram. By default, the endpoint uses port `5590`; however, it can be configured to use another port.

Reviewer: ```suggestion
EDS requires an endpoint to connect to REST APIs on the local device, which is shown outlined in blue in the diagram. By default, the endpoint uses port `5590`; however, you can configure it to use another port.
```
